src/config/config.rs

```diff
@@ -387,6 +390,23 @@ pub struct Http {
   pub group_by: Vec<String>,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+pub enum ExprNode {
```

Comment:
```suggestion
pub enum ExprBody {
```

Reply:
I renamed it to Node because when we do if-expressions we'd need the same enum, something like
```rust
If {
  cond: Box<ExprNode>,
  then: Box<ExprNode>,
  els: Box<ExprNode>
}
```

Reply:
Renamed it back to `ExprBody`.